docs/troubleshoot/app-framework/zss-error-codes.md

```diff
@@ -0,0 +1,371 @@
+# ZSS Error Message Codes
+
+The following error message codes may appear on ZSS log. Use the following message code references and the corresponding reasons and actions to help troubleshoot issues.
+
+## ZSS informational messages
+
+### ZWES0013I
+
+  ZSS Server has started. Version '%s'
+
+  **Reason:**
+
+  ZSS Server has started.
+
+  **Action:**
+
+  No action required.
+
+### ZWES0014I
+
+  ZIS status - '%s' (name='%.16s', cmsRC='%d', description='%s', clientVersion='%d')
+
+  **Reason:**
+
+  The message shows status of the connection to Privileged Server: ZIS status - `<OK or Failure>` (name=`<Privileged Server Name>`, cmsRC=`<RC>`, description=`<description`, clientVersion=`<version>`)
+
+  **Action:**
+
+  if Status is `OK` then no action required. If Status is `Failure` see [ZLUX unable to communicate with zssServer](app-troubleshoot/#zss-server-unable-to-communicate-with-x-mem).
+### ZWES0035I
+
+  ZSS Server settings: Address='%s', port='%d', protocol='%s'
+
+  **Reason:**
+
+  Server is starting using Address=`<IP address>`, port=`<port>`, protocol=`http` or `https`
```

Comment:
Note: http can still be https if attls=true?

Reply:
right